uuu/fat_write.lst

```diff
@@ -7,5 +7,6 @@ uuu_version 1.1.4
 
 FB: ucmd setenv fastboot_buffer ${loadaddr}
 FB: download -f _image
+FB: ucmd if test -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize;
```

Comment:
where set $filesize environment?

Reply:
I found it. latest uboot fastboot set filesize

Reply:
@nxpfrankli sorry I make a mistake on script. I need to fix it out

Reply:
ucmd if test -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize; fi

Reply:
FB: ucmd if test ! -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize; fi